teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>Prepare the protein ligand comples -&gt; comple<strong>x</strong></p><p> </p><p>Suggestion: I would move the 'Merge protein and ligand' one level lower: </p><ul><li>Prepare the protein ligand complex</li><li class="ql-indent-1">Prepare protein</li><li class="ql-indent-1">Prepare ligand</li><li class="ql-indent-1">Merge protein and ligand</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>